keras/backend/jax/distribution_lib_test.py

```diff
@@ -0,0 +1,175 @@
+"""Test for Jax backend distribution_lib.py."""
+
+import os
+
+import jax
+import numpy as np
+import pytest
+
+from keras import backend
+from keras import layers
+from keras import models
+from keras import testing
+from keras.backend import distribution_lib as backend_dlib
+from keras.distribution import distribution_lib
+
+
+@pytest.mark.skipif(
+    backend.backend() != "jax",
+    reason="Backend specific test",
+)
+class JaxDistributionLibTest(testing.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Due to https://github.com/google/jax/issues/17188, we can't
```

Comment:
Btw, I think this need to be on the top level (at the import time), otherwise it will be too late for jax to pickup the setting.

Reply:
Done